suite-common/message-system/config/config.v1.json

```diff
@@ -439,14 +348,14 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<=24.6.3",
+                        "desktop": "<=24.10.1",
```

Comment:
This is kind of drastic as it is only 1 month old Suite. The reasoning behind is that the 24.10.1 contains bug in send form that is already fixed in 24.11.3 and we want users to migrate quickly,